### PR TITLE
增加刷卡支付基础功能

### PR DESCRIPTION
**Pull request by jeremylam**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request
- Commit: acf33a0438cd8c8d324b5e29af6134b5347d698d
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 9aff877d993024cded0048a206bd0517fbb3f920
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 27416d37381e00646cf91612d704592ebab1d08c
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 4b7209e72db7478ebcdb7e6f6d6d91cfbea257c2
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: ab83459f477930be3a5fab36c19132033e46a911
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 76ad80368e3f90704dfb8bcb646ddb577c5d4400
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: d7605e65217eaee61ce3294bfaefa7677d764f65
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/jxtech/wechatpy/blob/master/CONTRIBUTING.md

---

This message was auto-generated by https://gitcop.com

**Comment by messense** [comment]
谢谢，稍后我会 review 您的代码。

发自我的 iPhone

File: wechatpy/pay/api/micropay.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            'attach': attach,
+            'out_trade_no': out_trade_no,
+            'total_fee': total_fee,
+            'fee_type': fee_type,
+            'spbill_create_ip': client_ip or get_external_ip(),
+            'goods_tag': goods_tag,
+            'limit_pay': limit_pay,
+            'auth_code': auth_code,
+        }
+        print data
```
Comment (messense) [inline wechatpy/pay/api/micropay.py]: 请删除掉 `print data` 这一行，首先在写库的时候不推荐使用 print 打印数据，另外，print 这样的写法在 Python 3 下不合法。

File: wechatpy/pay/api/micropay.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        return self._post(r'pay/micropay', data=data)
```
Comment (messense) [inline wechatpy/pay/api/micropay.py]: 这里为什么要用 `r'pay/micropay` 呢，直接用普通字符串就好了。

**Comment by GitCop** [comment]
There were the following issues with your Pull Request
- Commit: acf33a0438cd8c8d324b5e29af6134b5347d698d
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 9aff877d993024cded0048a206bd0517fbb3f920
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 27416d37381e00646cf91612d704592ebab1d08c
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 4b7209e72db7478ebcdb7e6f6d6d91cfbea257c2
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: ab83459f477930be3a5fab36c19132033e46a911
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 76ad80368e3f90704dfb8bcb646ddb577c5d4400
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: d7605e65217eaee61ce3294bfaefa7677d764f65
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 206f71cfcddf7cafd36349251b44ff527e3310b5
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/jxtech/wechatpy/blob/master/CONTRIBUTING.md

---

This message was auto-generated by https://gitcop.com

**Comment by jeremylam** [comment]
谢谢你的指正。

**Comment by messense** [comment]
Thanks!

**Pull request merged**